src/constants/connections/integrationVariablesMapping.constants.ts

```diff
@@ -3,16 +3,17 @@ import { Integrations } from "@src/enums/components";
 export const integrationVariablesMapping = {
 	[Integrations.github]: {
 		pat: "pat",
-		secret: "Secret",
+		secret: "pat_secret",
 	},
 	[Integrations.slack]: {
 		bot_token: "botToken",
 		app_token: "appToken",
 	},
 	[Integrations.twilio]: {
 		account_sid: "AccountSID",
-		api_key: "apiKey",
-		api_secret: "apiSecret",
+		api_key: "Username",
+		api_secret: "Password",
+		auth_token: "Password",
```

Comment:
We don't have `auth_token` in twilio.

Reply:
we have 
![image](https://github.com/user-attachments/assets/232c304b-4f51-40ea-80d0-3bb7025c3443)
![image](https://github.com/user-attachments/assets/dcf6ed74-1974-4a5f-91de-34dbe5fda434)

Reply:
I get it now, my bad.